### ISSUE TITLE
Bert_model doesn't work

### ISSUE THREAD

**Issue by geogreff** [brightmart/bert_language_understanding#7]
Hi, I tried your bert_model rather than bert_cnn_model. Bert_model could get about 75% F1 score on language model task. But using the pretrained bert_model to finetune on classification task, it didn't work. F1 score was still about 10% after several epoches. It is something wrong with bert_model?

**Comment by brightmart** [comment]
by default in step 1 and step 2, we are using bert_cnn_model. for bert_model, it has a bug need to fix. we suspect it is may related to the way we handle k,v,q, or some other problem. can you check and do something work on it?

**Comment by geogreff** [comment]
According to the paper, pretraining loss should be sum of the masked LM likelihood and next sentence prediction likehood. Your bert_model implementation just consists of masked LM likelihood. So, it perform well on LM task but the followed classification task.
